modules/core/test/parse-file/auto-parse.spec.js

```diff
@@ -4,7 +4,7 @@ import {parseFileSync} from '@loaders.gl/core';
 import {OBJLoader} from '@loaders.gl/obj';
 import {KMLLoader} from '@loaders.gl/kml';
 
-import KML from '@loaders.gl/kml/../data/KML_Samples.kml';
+import KML from '../../../kml/data/KML_Samples.kml';
```

Comment:
Why? This is a lot more fragile. I thought we agreed that the alias system was OK as long as we kept it inside loaders.gl?

As far as I can tell it is still used in other files, you just removed it in one place?

Reply:
There are two alias systems: one is used with `module-alias` and automatically created by ocular, which affects all import/require usages. The alias can point to different locations depending on test mode (src or dist).

The second alias systems is still in this repo, always point to src, and only affect `readFile`.

This is the former case, in which using relative path from an alias will break one of the two test modes.